### PR TITLE
Add resiliency gtfs read

### PR DESCRIPTION
**Pull request by patochectp**

### CODE REVIEW

**Comment by woshilapin** [comment]
Do you think we need tests for this? Note that I'm saying that but I'm not even sure how to test for logging... :wink:

File: src/gtfs/read.rs [hunk 1]
```diff
@@ -607,7 +610,14 @@ where
             let mut rdr = csv::Reader::from_reader(reader);
             let mut transfers = vec![];
             for transfer in rdr.deserialize() {
-                let transfer: Transfer = transfer.with_context(ctx_from_path!(path))?;
+                let transfer: Transfer = match transfer {
```
Comment (datanel) [inline src/gtfs/read.rs]: you can use skip_fail!
```rust
let transfer: Transfer = skip_fail!(transfer.with_context(ctx_from_path!(path)));
```

Comment (patochectp) [reply src/gtfs/read.rs]: skip_fail! handle "with_context" error and not deserialize error

Comment (datanel) [reply src/gtfs/read.rs]: skip_fail does exactly what you do with match + warn!

**Pull request merged**